spec/features/idv/analytics_spec.rb

```diff
@@ -867,6 +867,8 @@
         to receive(:service_provider_homepage_url).and_return(return_sp_url)
       allow_any_instance_of(Idv::InPerson::ReadyToVerifyPresenter).
         to receive(:sp_name).and_return(sp_friendly_name)
+      allow(IdentityConfig.store).to receive(:in_person_proofing_enforce_tmx).
+        and_return(false)
```

Comment:
Gina to do: change to true- the state we are going into and update assert values

Reply:
Completed with commit c7116a3dbcbca44128a913974180cc21249eb0b0